### DISCUSSION TITLE
Coverart picture in Music Folder

### DISCUSSION THREAD

**Discussion by Gt1ult** [LMS-Community/slimserver#709] [Ideas]
Dear Community,

Please could you advice if there is a way to see the cover picture in the folder when I navigate my music using "Music Folder"?

Furthermore do you know if it is possible to change view order of the files by showing first the folder in alphanumeric order and followed by single files in alphanumeric order?

It could really help my navigation experience, in fact I am using a lot "Music Folder".

Thanks!!

**Comment by CornelisJ** [comment]
FYI, I have my music folder organized in such a way that I have a separate folder for each album (My Music => Artist => Album Name). In each album folder I have a single (hidden) cover art picture called Folder.jpg. When I browse my Music Folder in LMS (using default skin), I get to see the covert art picture after I enter the album folder. This is shown in attached screenshot. Hopefully this answers your question.

![LMS_Music_Folder_cover_art](https://user-images.githubusercontent.com/59809764/139349465-c2989121-d02c-4bb3-b552-c2f88e9579eb.JPG)
